src/common/src/util/epoch.rs

```diff
@@ -151,8 +156,8 @@ impl EpochPair {
     }
 
     pub fn inc(&mut self) {
-        self.curr += 1;
-        self.prev += 1;
+        self.prev = self.curr;
+        self.curr += EPOCH_INC_MIN_STEP_FOR_TEST;
```

Comment:
If `inc` is not used only in test, then the constant should not contain the `_FOR_TEST` suffix?

Reply:
`inc` is only used in test, in real scenarios, inc will not be used in next epoch.